lib/trie/version.go

```diff
@@ -18,27 +28,90 @@ const (
 	// V0 is the state trie version 0 where the values of the keys are
 	// inserted into the trie directly.
 	// TODO set to iota once CI passes
-	V0 Version = 1
+	V0 Version = iota
+	V1
```

Comment:
Do you think it makes sense to mirror the parity implementation and name this `Layout`? I think it makes more sense given that the underlying db isn't changed, so it's not like we're versioning what is stored.

Reply:
Sure!